comps/comps-foreman-el8.xml

```diff
@@ -461,8 +461,8 @@
       <packagereq type="default">rubygem-hirb-unicode-steakknife-doc</packagereq>
       <packagereq type="default">rubygem-hocon-doc</packagereq>
       <packagereq type="default">rubygem-hoe-doc</packagereq>
-      <packagereq type="default">rubygem-httpclient-doc</packagereq>
```

Comment:
I think none of the changes in this file is actually relevant, mind reverting it?